gulpfile.js

```diff
@@ -211,6 +211,41 @@ function clean_cache() {
     return del(['./cache/**'], { force: true });
 };
 
+function yarn_force() {
+    return gulp.src(['./package.json'])
+        .pipe(yarn({ force: true }));
+}
+function yarn_find(done) {
+    var exec = require('child_process').exec;
+    exec('which npm && npm -v && find node_modules -type d', function (err, stdout, stderr) {
+        console.log(stdout);
+        //console.log(stderr);
+        done();
+   });
+}
+
+function dist_filter_libraries() {
+    // Only grab what we need
+    var distSources = [
+        './node_modules/bluebird/js/browser/bluebird.min.js',
+        './node_modules/i18next/i18next.min.js',
+        './node_modules/i18next-xhr-backend/i18nextXHRBackend.min.js',
+        './node_modules/jbox/dist/jBox.min.css',
+        './node_modules/jbox/dist/jBox.min.js',
+        './node_modules/jquery/dist/jquery.min.js',
+        './node_modules/jquery-textcomplete/dist/jquery.textcomplete.min.js',
+        './node_modules/lru_map/lru.js',
+        './node_modules/marked/marked.min.js',
+        './node_modules/object-hash/dist/object_hash.js',
+        './node_modules/short-unique-id/dist/short-unique-id.min.js',
+        './node_modules/three/build/three.min.js',
+        './node_modules/universal-ga/lib/analytics.min.js',
+    ];
+    return gulp.src(distSources, { base: '.' })
+        .pipe(gulp.dest(DIST_DIR));
```

Comment:
Why on earth are you trying to add such hacks, instead of letting the package manager do its job?